### PR TITLE
ADD: NamedEntities

### PR DESCRIPTION
**Pull request by sihrc**

### CODE REVIEW

File: src/io/indico/api/results/BatchIndicoResult.java [hunk 1]
```diff
@@ -57,28 +59,52 @@ public BatchIndicoResult(Api api, Map<String, ?> response) throws IndicoExceptio
...
     }
 
     @SuppressWarnings("unchecked")
+    public List<Map<String, Map<Category, Double>>> getNamedEntities() throws IndicoException {
+        if (!results.containsKey(Api.NamedEntities))
+            throw new IndicoException(Api.NamedEntities.name + " was not included in the request");
+
+        List<Map<String, Map<Category, Double>>> result = new ArrayList<>();
+
+        List<Map<String, Map<String, Object>>> responses = (List<Map<String, Map<String, Object>>>) results.get(Api.NamedEntities);
```
Comment (anneCarlson) [inline src/io/indico/api/results/BatchIndicoResult.java]: This either needs to be made a bit clearer (perhaps by pulling out some of the logic to a well named function) or deserves a small comment. It's not terribly obvious why this needs to be done.

Comment (Slater-Victoroff) [inline src/io/indico/api/results/BatchIndicoResult.java]: Ah, it's a Multi-API thing. There's probably a way to make this a nice aspect since it's in every call.

Comment (sihrc) [inline src/io/indico/api/results/BatchIndicoResult.java]: It's pulled out in a different PR actually.

**Comment by sihrc** [comment]
This is ready to merge again
